src/Microsoft.AspNetCore.Mvc.Razor/Internal/CompositeTagHelperDescriptorResolver.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Compilation.TagHelpers;
+using Microsoft.AspNetCore.Razor.Runtime.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Internal
+{
+    public class CompositeTagHelperDescriptorResolver : ITagHelperDescriptorResolver
+    {
+        public IList<TagHelperDescriptorResolver> _resolvers;
+
+        public CompositeTagHelperDescriptorResolver(
+            TagHelperDescriptorResolver tagHelperDescriptorResolver,
+            ViewComponentTagHelperDescriptorResolver viewComponentTagHelperDescriptorResolver)
+        {
+            _resolvers = new List<TagHelperDescriptorResolver>();
+            _resolvers.Add(tagHelperDescriptorResolver);
```

Comment:
Could we just call these two explicitly in `Resolve` rather than putting them in a list? It's probably terser too
`descriptors.AddRange(_tagHelperDescriptorResolver.Resolve(resolutionContext));`
`descriptors.AddRange(_viewComponentTagHelperDescriptorResolver.Resolve(resolutionContext));`